NEWS.md

```diff
@@ -88,6 +88,8 @@
 
 14. `by=...get()...` could fail with `object not found`, [#4873](https://github.com/Rdatatable/data.table/issues/4873) [#4981](https://github.com/Rdatatable/data.table/issues/4981). Thanks to @sindribaldur for reporting, and @OfekShilon for fixing.
 
+15. `as.IDate` works on character input where the first input is `''` by treating it as `NA`, [#4676](https://github.com/Rdatatable/data.table/issues/4676). This is consistent with how `''` is treated when it appears later in the input; the underlying inconsistency in `as.Date.character` was fixed in R version 4.0.3; we applied the change in `data.table` as well because we depend only on a much earlier version of R.
```

Comment:
The words 'works' needs replacing with detail. Overall the news item could be maybe half as long. The skeleton that springs to mind is something like
> `as.IDate(c('',somedate))` now returns ... instead of .... This is a backport of rXXX from R 4.0.3 so that we can continue to support and test data.table on R 3.1.0+.

Reply:
I suggested `as.IDate(c('',somedate))` like that but I don't see that.  I see 'error' in the edit but not what the error was. And the news item is still long.  No worries - I'll have a go then.